src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs.cljs

```diff
@@ -99,9 +101,12 @@
                                             (react/call :load-method-configs this)
                                             (swap! state update-in [:method-confs]
                                               assoc (:selected-index @state) new-conf)))}]
-         (:method-confs-loaded? @state) [WorkspaceMethodsConfigurationsList {:workspace (:workspace props)
-                                                                             :method-confs (:method-confs @state)
-                                                                             :parent-state state}]
+         (:method-confs-loaded? @state) [WorkspaceMethodsConfigurationsList
+                                          {:on-import
+                                             (fn [] (react/call :load-method-configs this))
```

Comment:
I guess since it was mock data, that actually importing anything wasn't terribly important.  I had never thought to add a mock MC to a list of mock MCs in a mock workspace.  I thought that refreshing the list was the important thing to do